modules/rubiconBidAdapter.js

```diff
@@ -472,4 +475,9 @@ export function resetUserSync() {
   hasSynced = false;
 }
 
+function isNaN(value) {
+  // eslint-disable-next-line no-self-compare
+  return value !== value;
```

Comment:
how does this work?

Reply:
I got it from here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN#Polyfill